config.hh.cmake

```diff
@@ -26,10 +26,11 @@
 # define @LIBRARY_NAME@_CONFIG_HH
 
 // Package version (header).
+# define @LIBRARY_NAME@_VERSION_UNKNOWN_TAG // Used to mention that the current version is unknown.
```

Comment:
@jcarpent thank you for the work you do for the community. However, could you please be less quick in merging such changes ?
Those change should be discussed. I strongly disapprove the idea of having this into the C++ code. Version computation **must** be handled at configure time.
If git does not fulfill all the requirement to compute the current version, then we should think of another method.

Reply:
This fix allows us to provide Pinocchio on ROS for instance. We had serious limitations due to ROS build, and in this way, we were able to handle it. The problem of computing the version is that currently, we do not have a common consensus in where to store it, knowing that the file `.version` is only provided in Releases, and it is not inside the package.

Maybe for each package, we should have something like package.xml which provided useful information like main contributors, website, etc.

I'm not against changing the strategy in future, but I would like the packages remain available for the community.

Reply:
> The problem of computing the version is that currently, we do not have a common consensus in where to store it

and we should solve that before doing so many changes. If you want a temporary fix in Pinocchio before this consensus is found, just set the version explicitly in the CMakeLists.

Reply:
It will appear in other places that have copied pasted the code from Pinocchio. So this is a temporay fix I agree, but will avoid other burden outside ;)

Reply:
A very simple temporary fix would have been `SET(PROJECT_VERSION ...)` in Pinocchio and EigenPy. There was no need to dirty the repo.